ui/src/index.html

```diff
@@ -1,6 +1,7 @@
-<html>
-
+<html lang="en">
```

Comment:
should we have a <!doctype html> as well?

Reply:
Yes, ofc.. fixed.